sdk/quantum/quantum/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@azure/quantum",
+  "author": "Microsoft Corporation",
+  "description": "Azure Quantum REST API client",
+  "version": "1.0.0-beta.1",
+  "dependencies": {
+    "@azure/core-paging": "^1.1.1",
+    "@azure/core-http": "^1.2.0",
+    "tslib": "^2.0.0"
+  },
+  "keywords": [
+    "node",
+    "azure",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "license": "MIT",
+  "main": "./dist/quantum.js",
+  "module": "./esm/index.js",
+  "types": "./esm/index.d.ts",
+  "devDependencies": {
+    "typescript": "4.1.2",
+    "rollup": "^1.16.3",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "uglify-js": "^3.4.9",
+    "@microsoft/api-extractor": "7.7.11",
+    "mkdirp": "^1.0.4"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Azure/azure-sdk-for-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/**/*.js",
+    "dist/**/*.js.map",
+    "dist/**/*.d.ts",
+    "dist/**/*.d.ts.map",
+    "esm/**/*.js",
+    "esm/**/*.js.map",
+    "esm/**/*.d.ts",
+    "esm/**/*.d.ts.map",
+    "./src//**/*.ts",
+    "README.md",
+    "rollup.config.js",
+    "tsconfig.json"
+  ],
+  "scripts": {
+    "build": "tsc && rollup -c rollup.config.js && mkdirp ./review &&  npm run extract-api",
+    "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/quantum.js.map'\" -o ./dist/quantum.min.js ./dist/quantum.js",
+    "prepack": "npm install && npm run build",
+    "clean": "rimraf dist dist-esm dist-browser test-dist test-browser types *.tgz *.log",
+    "extract-api": "api-extractor run --local",
+    "build:samples": "echo skip",
+    "build:node": "echo skip",
+    "build:browser": "echo skip",
+    "build:test": "echo skip",
+    "build:test:node": "echo skip",
+    "build:test:browser": "echo skip",
+    "check-format": "echo skip",
+    "coverage": "echo skip",
+    "execute:samples": "echo skip",
+    "format": "echo skip",
+    "lint:fix": "echo skip",
+    "lint": "echo skip",
+    "prebuild": "echo skip",
+    "pack": "echo skip",
+    "swagger": "echo skip",
+    "integration-test": "echo skip",
+    "integration-test:browser": "echo skip",
+    "integration-test:node": "echo skip",
+    "test": "echo skip",
+    "test:browser": "echo skip",
+    "test:node": "echo skip",
+    "unit-test": "echo skip",
+    "unit-test:browser": "echo skip",
+    "unit-test:node": "echo skip",
+    "docs": "echo skip"
```

Comment:
any reason we cannot fill the above scripts like we have for other packages rather than keeping them as `echo skip`?

Reply:
I see that you answered in https://github.com/Azure/azure-sdk-for-js/pull/13345#discussion_r562818648, I have replied there as well :)

Reply:
The CI actually tries to run them and without actual tests they will fail. Also, we need to ensure each script runs before proceeding forward. Since this is a first commit, I made it to skip